.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish (latest)
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    name: Build and publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: pnpm/action-setup@v2.4.0
+        with:
+          version: 8
+
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: 'https://registry.npmjs.org'
+          node-version: 16
+          cache: 'pnpm'
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Version
+        run: pnpm version --no-git-tag-version ${{ github.event.release.tag_name }}
+
+      - name: Publish
+        run: |
+          pnpm config set "//registry.npmjs.org/:_authToken" "${NPM_TOKEN}"
```

Comment:
```suggestion
```
Not needed, it's already done via 
```
      - uses: actions/setup-node@v3
        with:
          registry-url: 'https://registry.npmjs.org'
```

Reply:
`setup-node` one, unfrotinately, is not picked up by `pnpm`

Reply:
That's very weird, I don't see why that would not be picked up